guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,196 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Watson Work Services gives your App two patterns for dealing with events, which take place either due to changes in the data model and/or due to actions humans take using a participating client in Watson Work Services such as the Workspace clients. One of these is called **Implicit Action Handler**, the other is called **Explicit Action Handler**.
+
+We are standardizing patterns in our programming model and thus the existing App Configuration flow, though is being kept for backwards compatibility, is being incorporated into this new ClientActionHandler flow. It will introduce a change to how to make your app configurable. Currently, in the [Your Apps section](https://developer.watsonwork.ibm.com/apps) there is a section called **Additional Information** where you enter a configuration URL (for configuring your app in a space) and Terms of Services for your app. With this change we will move Terms of Service to it's own section, and remove the **Additional Information** section. The rest of this guide explains what you will do to set up your app to handle user actions regarding App configuration. If you had already establised an app with configuration, you will not need to do anything to continue working as is, and can update your app as needed. 
+
+There are three main concepts to Actions
+
+| Concept                       | Description       |
+| ----------------------------- |:------------------|
+| **Declare**                   | The developer will declare their app will handle an action, either **/Commands** or **Actions from detected intents** or **Action Fulfillment UI**. |
```

Comment:
This part i missing from the "declare" section:  in addition to declaring that an app can handle /cmds or actions from intents or actions from Action Fulfillment flows the developer can also declare the app can handle recognized actions that a client may implement such as:
- configuring an application
- sharing a resource into the conversation
- and many others to come

Reply:
Suggest something such as:

The developer will declare their app will handle an action via various constructs provided in the developer user experience such as: 
- **/Commands** that trigger an Action Fulfillment experience
- **Cognitively detected actions** from Watson Assistant training
-  **Action Fulfillment UI** elements such as buttons, etc
- **Client provided** actions such as App Configuration and Sharing Resources